_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,206 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements within a given [document context](#document-context) and [exposed to assistive technologies](#exposed-to-assistive-technologies) that have a [non-empty](#non-empty) accessible name.
```

Comment:
You can remove the "within a given document context". That stuff doesn't come into play until the expectation.